.github/workflows/build_and_test.yaml

```diff
@@ -52,16 +50,33 @@ jobs:
         files: ./coverage.xml
         name: codecov-envoy-gateway
         verbose: true
+  
+  conformance-tests:
+    runs-on: ubuntu-latest
+    needs: [unit-tests]
+    strategy:
+      matrix:
+        kind_node_tag: ["v1.24.0", "v1.23.6", "v1.22.9"]
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ./tools/github-actions/setup-deps
+    - name: Run Conformance Tests
+      env:
+        KIND_NODE_TAG: ${{ matrix.kind_node_tag }}
+        CONFORMANCE_UNIQUE_PORTS: false
+      run: make conformance
+
+
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    needs: [conformance-tests]
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ./tools/github-actions/setup-deps
 
-    # build
     - name: Build Multiarch EG Binaries
```

Comment:
I think order matters here, we build the binary and then copy it into the image

Reply:
I'm pretty sure that `make conformance` ends up rebuilding the binary even if it already exists locally, but I'll have to check again tmrw.

Reply:
nvm it should be fine